forum/forum/settings.py

```diff
@@ -137,3 +140,62 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.IsAuthenticated',
+    ],
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
+    ),
+    'DEFAULT_THROTTLE_CLASSES': [
+        'rest_framework.throttling.AnonRateThrottle',
+        'rest_framework.throttling.UserRateThrottle'
+    ],
+    'DEFAULT_THROTTLE_RATES': {
+        'anon': '100/day',
+        'user': '1000/day',
+        'token_obtain': '500/hour',
+        'token_refresh': '100/hour',
+    },
```

Comment:
Throttle rates are hardcoded, which might not be flexible enough for different environments. Use environment variables for throttle rates to make them configurable:
```
'DEFAULT_THROTTLE_RATES': {
    'anon': environ.get('ANON_THROTTLE_RATE', '100/day'),
    'user': environ.get('USER_THROTTLE_RATE', '1000/day'),
    'token_obtain': environ.get('TOKEN_OBTAIN_RATE', '500/hour'),
    'token_refresh': environ.get('TOKEN_REFRESH_RATE', '100/hour'),
},
```

Reply:
Done